dockerfiles/dev/Dockerfile

```diff
@@ -21,7 +22,20 @@ RUN chmod +x /usr/local/bin/dumb-init
 COPY gdn/gdn-* /usr/local/concourse/bin/gdn
 RUN chmod +x /usr/local/concourse/bin/gdn
 
-# add fly executables
+# containerd executables
+ARG RUNC_VERSION=v1.0.0-rc8
+ARG CNI_VERSION=v0.8.2
+ARG CONTAINERD_VERSION=1.3.0
+
+RUN curl -sSL https://github.com/containerd/containerd/releases/download/v$CONTAINERD_VERSION/containerd-$CONTAINERD_VERSION.linux-amd64.tar.gz \
+      | tar -zvxf - -C /usr/local/concourse/bin --strip-components=1
```

Comment:
this might look wrong (in the sense that we're not `mkdir -p /usr/local/concourse/bin` before), but that's because the previous step (`COPY gdn ...`) already creates that tree.